docs/src/main/asciidoc/security-identity-providers.adoc

```diff
@@ -9,18 +8,18 @@ include::_attributes.adoc[]
 :diataxis-type: concept
 :categories: security
 
-In the Quarkus Security framework, identity providers play a key role in authentication and authorization, providing services for storing and verifying user identities.
-The Jakarta Persistence `IdentityProvider` creates a `SecurityIdentity` instance used during user authentication to verify and authorize access requests, making your Quarkus application secure.
+In the Quarkus Security framework, identity providers play a crucial role in authentication and authorization, providing services for storing and verifying user identities.
```

Comment:
This text is modifief in another PR so there will be a merge conflict here

Reply:
@sberyozkin - Self-inflicted conflict fixed. The identity providers doc updates over on my other branch are now out of this branch and this is rebased to main. Thanks for flagging. I'll double-check this in the am with fresh eyes.